api/Dockerfile

```
@@ -1,5 +1,5 @@
 # * https://turbo.build/repo/docs/handbook/deploying-with-docker
-FROM node:18.16.0-alpine AS base
+FROM node:18.20.1-alpine AS base
```

```suggestion
FROM node:18.20-alpine AS base
```
plutôt `node:18.20-alpine`, qu'en pensez vous @pitiscarf @naustra